packages/renderer/src/lib/pod/PodsList.svelte

```diff
@@ -146,6 +146,7 @@ let containersColumn = new TableColumn<PodInfoUI>('Containers', {
 
 let ageColumn = new TableColumn<PodInfoUI, Date | undefined>('Age', {
   renderer: TableDurationColumn,
+  comparator: (a, b): number => new Date(a.created).getTime() - new Date(b.created).getTime(),
```

Comment:
nit: comparator is not covered by tests. There is a test for button to appear, but no test for comparator actually sorts records in correct order.

Reply:
:+1:

Reply:
Thanks for adding a test for it. Resolving the conversation.